functions/graphql/index.js

```diff
@@ -76,7 +76,12 @@ function configureServer() {
     context: async ({ req }) => {
       const token = (req.headers.authorization || "").replace("Bearer ", "");
       // If you need to test a specific user's graphQL abilities
+
+      // FOR DEBUGGING, COMMENT OUT IN PRODUCTION
```

Comment:
A better way to do this would be to put it behind `process.env.NODE_ENV === 'production'`. It automatically ignores that code in a production environment.